stdlib/copy.pyi

```diff
@@ -11,6 +12,10 @@ PyStringMap: Any
 def deepcopy(x: _T, memo: dict[int, Any] | None = None, _nil: Any = []) -> _T: ...
 def copy(x: _T) -> _T: ...
 
+if sys.version_info >= (3, 13):
+    def replace(obj: _T, /, **changes: Any) -> _T: ...
+    __all__ += ["replace"]
```

Comment:
See us mostly using `+=` in most places with `__all__`, guessing this is for better static analysis support, or something like that?

Reply:
Yes, this is fine, it's so we don't have to repeat the whole list for every version.

Reply:
Oh, I meant versus using `append`.

Reply:
Except it actually isn't in `__all__`: https://github.com/python/cpython/blob/89bf33732fd53fbb954aa088cfb34f13264746ad/Lib/copy.py#L59 . So we shouldn't add it in the stub either.

This may be an oversight, so consider opening an issue over on CPython for adding replace to `copy.__all__`. But as long as it's not there at runtime, we shouldn't add it in the stub either.

Reply:
My bad, let me fix that.

Reply:
https://github.com/python/typeshed/pull/12262/commits/775e88b1fa16cdea46b9ac7ab8930dbd08369848